pyRF24/setup.py

```diff
@@ -74,33 +73,13 @@
 os.environ["CFLAGS"] = cflags
 
 # get the proper boost.python lib symlink name according to version of python
-if version_info >= (3,):
-    BOOST_LIB = "boost_python3"
-else:
-    BOOST_LIB = "boost_python"
-
-crossunixccompiler.register()
-
-long_description = """
-.. warning:: This python wrapper for the RF24 C++ library was not intended
-    for distribution on pypi.org. If you're reading this, then this package
-    is likely unauthorized or unofficial.
-"""
+BOOST_LIB = "boost_python" + (
+    "" if version_info < (3,) else "%d%d" % (version_info.major, version_info.minor)
+)
```

Comment:
The reusable CI workflow needed updating because `libboost-python-dev` is built against whatever the system's preferred python version is. All is good now. We can still build older version of the py wrappers in CI (if not cross-compiling the lib).